### PR TITLE
added services and resolvers for prompts/comments/songsSubs

### PR DESCRIPTION
**Pull request by kengwall8**
just added basic services and resolvers, wasn't totally sure about comment and song submissions schemas so I added full users/song types for their respective fields

### CODE REVIEW

**Comment by kengwall8** [comment]
some errors actually dont approve yet

File: src/server/schemas/Prompt.ts [hunk 1]
```diff
@@ -16,6 +16,9 @@ export class Prompt {
     @Field()
     prompt: string;
 
+    @Field()
+    posted_by: User
```
Comment (celinanperalta) [inline src/server/schemas/Prompt.ts:20]: change to string for user_id

File: src/server/services/songSubmissionService.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import client from "../config/esConnection";
+import { Song } from "../schemas/Song";
+import { SongSubmission } from "../schemas/SongSubmission";
+import { User } from "../schemas/User";
+import {v4 as uuid} from 'uuid';
+
+
+const getSongSubmissions = async () : Promise<SongSubmission[]> => {
+    const {body} = await client.search({
+        index: "songsubmissions",
```
Comment (celinanperalta) [inline src/server/services/songSubmissionService.ts:10]: nitpick: rename `songsubmissions` to `songSubmissions`

**Review by celinanperalta — APPROVED** [review]
Overall good, let's try to prefer camelcase over snakecase and then go ahead and push

File: src/server/services/songSubmissionService.ts (hunk 2 above)
Comment (kengwall8) [reply src/server/services/songSubmissionService.ts:10]: indices have to be all lowercase :/

Comment (celinanperalta) [reply src/server/services/songSubmissionService.ts:10]: oh rip

**Pull request merged**